requirements.txt

```diff
@@ -0,0 +1,32 @@
+backports-datetime-fromisoformat==1.0.0
+betterproto==1.2.5
+black==21.8b0
```

Comment:
This seems like it's pulling in a lot of extra stuff
betterproto: only used in examples (separate venv? I'd say defer this and just don't add their deps)
black: for sure we don't use (but we should!)
pyperclip: random useful dev tool?

Is this just stuff that you happen to have used in a venv locally?

Is there a way we can get this down to mako + schema (and their deps)?

Reply:
Yeah, "a bunch of random stuff" is a good summary. I started clean and just installed mako and schema and shrunk this a lot.